superset-frontend/src/components/ReportModal/HeaderReportDropdown/index.test.tsx

```diff
@@ -0,0 +1,159 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import userEvent from '@testing-library/user-event';
+import { render, screen } from 'spec/helpers/testing-library';
+import * as featureFlags from 'src/featureFlags';
+import { FeatureFlag } from '@superset-ui/core';
+import HeaderReportDropdown from '.';
+
+let isFeatureEnabledMock: jest.MockInstance<boolean, [string]>;
+
+interface HeaderReportProps {
```

Comment:
shouldn't we reference this in a types.tsx file?

Reply:
can we move it to the types file? i think it will be better served there vs. duplicating code

Reply:
I exported it in the main component, I can create a type.ts file, but it would only hold this one interface.